src/buza/urls.py

```diff
@@ -8,6 +8,8 @@
 
 
 urlpatterns = [
+    path('', generic.RedirectView.as_view(pattern_name='questions'), name='home'),
```

Comment:
should we maybe check if a user is logged in to redirect them to login if arent because we extract the home questions depending on their grade or the boards that they have joined

Reply:
We can probably do this in follow-up, as we improve the views: for now, I just wanted the `/` URL not to 404.